docs/modules/ROOT/pages/advanced-how-tos/releasing/create-release.adoc

```diff
@@ -55,4 +55,4 @@ $ kubectl apply -f Release.yaml -n dev
 . Click on the application that is being released
 . Click on the *Releases* tab
 . See the recent releases that have been created for the application.
-. You can find a link to the release pipeline run by clicking on the name of the release that you created.
\ No newline at end of file
+. You can find a link to the release pipeline run by clicking on the name of the release that you created.
```

Comment:
What is this sorcery, what is this change? Two completely same lines are being removed and added? I don't understand, pls help.

Reply:
I had the same confusion. I think this was because of VS code adding a new line at the end (I saw it locally in my IDE), which however is not visible here in GitHub as a new line idky :peepo-shrug: